nilearn/_utils/niimage.py

```diff
@@ -0,0 +1,127 @@
+import copy
```

Comment:
Can you give us a docstring that says what's in this module?

Also license information as a comment.